hotel-management/src/layouts/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import Avatar from '@/assets/images/avatar.svg?react';
+import ToggleMenuIcon from '@/assets/images/toggle-menu.svg?react';
```

Comment:
Why file extesion is `svg?react`, not make any sense

Reply:
because I installed 'vite-plugin-svgr' to allow me to import SVG files as React components, and it requires '?react' in the end of the path.